src/Item.js

```diff
@@ -179,9 +179,13 @@ class Item extends EventEmitter {
 
     // if the item was saved with a custom inventory hydrate it
     if (this.inventory) {
+      // handle equipped containers.
+      if (!(this.inventory instanceof Inventory)) {
```

Comment:
1. Add a wearable container to the items.yml file. For instance:
```
- id: 16
  name: "Ragged Canvas Knapsack"
  type: CONTAINER
  roomDesc: "Ragged Canvas Knapsack"
  keywords: [ "rag", "ragged", "knap", "knapsack", "canvas" ]
  description: "A small canvas knapsack attached to a shoulder strap. The strap appears frayed."
  items: [ ]
  maxItems: 3
  slot: 'back'
  quality: common
  properties:
    stats:
      armor: 1
  behaviors:
    sellable:
      value: 2
      currency: gold
```

2. Put said item in a room or container via editing the .yml
3. Log in to a character, acquire the equippable-container-item, put something into it, equip it, save, and log off.
4. Log back in and see if the server crashes immediately due to trying to call hydrate on a plain JS object.